packages/backend/src/ee/services/ai/tools/runQuery.ts

```diff
@@ -115,6 +116,15 @@ export const validateRunQueryTool = (
         queryTool.customMetrics,
         queryTool.tableCalculations,
     );
+
+    // Validate table calculations
+    validateTableCalculations(
+        explore,
+        queryTool.tableCalculations,
+        queryTool.queryConfig.dimensions,
+        queryTool.queryConfig.metrics,
+        queryTool.customMetrics,
```

Comment:
I'm a bit confused, wasn't this validation here before? 🤔

Reply:
That's a bug!